classes/controller/admin/form.ctrl.php

```diff
@@ -46,9 +46,15 @@ public function before_save($item, $data)
 
         // The field data is serialized in json (cf. Pull Request #15)
         $fields_post = \Input::post('fields', null);
-        $fields_data = array();
-        if ($fields_post !== null) {
-            $fields_data = json_decode($fields_post, true);
+        if (empty($fields_post)) {
+            throw new \Exception('Error: empty fields data.');
```

Comment:
I thing the error message can be more friendly, for example "Your form seems to have no field." ?